doc/radosgw/dynamicresharding.rst

```diff
@@ -85,6 +85,54 @@ Bucket resharding status
 
    # radosgw-admin reshard status --bucket <bucket_name>
 
+The output is a json array of 3 objects (reshard_status, new_bucket_instance_id, num_shards) per shard.
+
+For example, the output at different Dynamic Resharding stages is shown below:
+
+``1. Before resharding occurred:``
+::
+
+  [
+    {
+        "reshard_status": "none",
+        "new_bucket_instance_id": "",
+        "num_shards": -1
+    }
+  ]
+
+``2. During resharding:``
+::
+
+  [
+    {
+        "reshard_status": "in-progres",
```

Comment:
"in-progress" has two "s"s

But the code looks correct. So perhaps this is just a typo in editing the example.

Reply:
Thanks, typo fixed.